src/components/styles/modal.less

```diff
@@ -28,10 +28,16 @@
         width: 40%;
         border-radius: 4px;
         box-shadow: 0px 2px 8px 0px rgba(0,0,0,0.16);
+        margin-bottom: 60px;
+
+        &.large {
+            width: 60%;
+            max-width: 1000px;
+        }
 
         // only applies for devices smaller than a desktop (defined by bootstrap)
         @media(max-width: 992px) {
-            width: calc(~'100% - 120px'); // 60px on each side (incl. top via padding)
+            width: calc(~'100% - 120px') !important; // 60px on each side (incl. top via padding)
```

Comment:
Why do we need the `!important` here? It should be ok to change the width by overriding these styles, right?